plugins/web_adapter/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Unreleased
 
-*None.*
+- Supports the WASM environment.
```

Comment:
Mention `pub upgrade (dio_web_adapter)` might be required for this update to take effect?

Reply:
Good idea, also could be added to the README too